NVActivityIndicatorView/NVActivityIndicatorPresenter.swift

```diff
@@ -249,9 +265,20 @@ public final class NVActivityIndicatorPresenter {
     private func hide() {
         guard let keyWindow = UIApplication.shared.keyWindow else { return }
 
+        let hasExitAnimation = self.enterExitAnimationDuration > 0
+
         for item in keyWindow.subviews
             where item.restorationIdentifier == restorationIdentifier {
-            item.removeFromSuperview()
+                if hasExitAnimation {
+                    UIView.animate(withDuration: 1, delay: 0, options: [.curveEaseOut], animations: {
```

Comment:
It seems `enterExitAnimationDuration` should be used here instead of hardcoded `1`.